extensions/wikia/AdEngine/js/wad/ilRecLoader.js

```diff
@@ -0,0 +1,33 @@
+/*global define*/
+define('ext.wikia.adEngine.wad.ilRecLoader', [
+	'ext.wikia.adEngine.adContext',
+	'ext.wikia.adEngine.utils.scriptLoader',
+	'wikia.document',
+	'wikia.window'
+], function (adContext, scriptLoader, doc, win) {
+	'use strict';
+
+	var wikiaApiController = 'AdEngine2ApiController',
+		wikiaApiMethod = 'getILCode';
+
+	function injectScript() {
+		var url = win.wgCdnApiUrl + '/wikia.php?controller=' + wikiaApiController + '&method=' + wikiaApiMethod;
+
+		scriptLoader.loadScript(url, {
+			isAsync: false,
+			node: doc.head.lastChild
+		});
+	}
+
+	function init() {
+		if (adContext.get('opts.babRec')) {
+			doc.addEventListener('bab.blocking', injectScript);
+		} else {
+			injectScript();
```

Comment:
Same question here - do we really want to inject this script without babDetection?

Reply:
And same answer: It's just an option. We had that before, why not leave it after cleanup-o-refactor. Waiting for grooming - we probably gonna remove that.

Reply:
Ok, removing.